lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -40,8 +40,9 @@ def _get_testable_interactive_backends():
     ]:
         reason = None
         missing = [dep for dep in deps if not importlib.util.find_spec(dep)]
-        if sys.platform == "linux" and not os.environ.get("DISPLAY"):
-            reason = "$DISPLAY is unset"
+        if (sys.platform == "linux" and not os.environ.get("DISPLAY") and
+                not os.environ.get("WAYLAND_DISPLAY")):
```

Comment:
should that be `if not `_c_internal_utils.display_is_valid()`?

Reply:
Yes, that works even better I think.